tests/main/interfaces-home/task.yaml

```diff
@@ -119,33 +123,37 @@ execute: |
 
     echo "============================================"
 
-    echo "When the plug is disconnected"
-    snap disconnect home-consumer:home
-    snap interfaces | grep -Pzq "$DISCONNECTED_PATTERN"
+    if ! snap forced-devmode ; then
```

Comment:
We can just exit 0 here as done in prior tasks, instead of indenting everything else.